server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -282,6 +286,8 @@ private void handleProvideEvidence(JSONObject evidence, Player player) {
 	  evidence.put(Constants.EVENT_TYPE, EventType.EVIDENCE_PROVIDED_NOTIFICATION);
 	  evidence.put(Constants.AUTHOR, player.getTag());
 	  suggester.sendEvent(evidence);
+	  handleSimpleEvent(suggester, EventType.PREVENT_SUGGEST);
+	  handleSimpleEvent(suggester, EventType.ALLOW_TURN_END);
```

Comment:
I forget my exact rationale, but I think it had to do with the player having to move before being able to make a suggestion, and suggestions move other players which causes some problems with allowing them to suggest before moving. So I just put everything serverside.